src/components/UploadVideo/UploadVideo.tsx

```diff
@@ -30,6 +30,7 @@ const UploadVideo = () => {
   useEffect(()=> {!ffmpeg.current.isLoaded() && load();}, []);
 
   const handleTranscodeClick = async (): Promise<void> => {
+    if (isTranscoding) return;
```

Comment:
Oooooh I see the previous bug, nice catch!

Reply:
user could abuse the button and computer would start running a lot of ffmpegs and overheating